app/src/main/java/ch/admin/bag/dp3t/checkin/events/EventsOverviewFragment.kt

```diff
@@ -0,0 +1,66 @@
+package ch.admin.bag.dp3t.checkin.events
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import ch.admin.bag.dp3t.R
+import ch.admin.bag.dp3t.checkin.generateqrcode.GenerateQrCodeFragment
+import ch.admin.bag.dp3t.checkin.generateqrcode.QRCodeViewModel
+import ch.admin.bag.dp3t.databinding.FragmentEventsOverviewBinding
+import org.crowdnotifier.android.sdk.model.VenueInfo
+
+class EventsOverviewFragment : Fragment() {
+
+	companion object {
+		fun newInstance(): EventsOverviewFragment {
+			return EventsOverviewFragment()
+		}
+	}
+
+	private lateinit var binding: FragmentEventsOverviewBinding
+	private val qrCodeViewModel: QRCodeViewModel by activityViewModels()
+
+	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+		super.onCreateView(inflater, container, savedInstanceState)
+		binding = FragmentEventsOverviewBinding.inflate(layoutInflater)
+		return binding.root
+	}
+
+	override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+		super.onViewCreated(view, savedInstanceState)
+
+		binding.eventsToolbar.setNavigationOnClickListener { requireActivity().supportFragmentManager.popBackStack() }
```

Comment:
All View setup can be done in `onCreateView()` instead, so we can do everything in one place and don't need to save the binding in a field.